src/main/java/math/Fraction.java

```diff
@@ -263,7 +263,12 @@ public Fraction reduce() {
     @Override
     public Object clone() {
         // TODO (T8192Z)
-        return null;
+        try {
+            return super.clone();
+        } catch (CloneNotSupportedException e) {
+            e.printStackTrace();
```

Comment:
The `CloneNotSupportedException` will be never thrown here, thus the lines
```
e.printStackTrace();
return null
```
should be removed. If you do not want the `catch` block to be empty, you still can use
```
throw new AssertionError`()
```